addons/evac/functions/fn_stretcher_attach.sqf

```diff
@@ -0,0 +1,213 @@
+_h = cursorObject;
```

Comment:
Let's get this file renamed to `fnc_stretchter_attach.sqf`

Reply:
And get a comment like this going and clean up the formatting so it's a bit more consistent and then I think should be good to go.
```
/*
 * Author: mharis001
 * Checks if AED Station can be performed on the patient.
 *
 * Arguments:
 * 0: Medic (not used) <OBJECT>
 * 1: Patient <OBJECT>
 *
 * Return Value:
 * Can CPR <BOOL>
 *
 * Example:
 * [player, cursorObject] call kat_circulation_fnc_AEDStationCondition
 *
 * Public: No
 */
```